cymulate-phishing-awareness/info.json

```diff
@@ -0,0 +1,304 @@
+      "description": "Retrieve the Phishing Awareness Campaign assessment history within the date range provided. If a date range is not provided, the response will retrieve the history from the last 30 days.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "fromDate",
+          "title": "From Date",
+          "type": "datetime",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify the date from where to fetch phishing awareness aampaign assessments .",
+          "description": "Specify the date from where to fetch phishing awareness campaign assessments ."
+        },
+        {
+          "name": "toDate",
+          "title": "To Date",
+          "type": "datetime",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify the date until which phishing awareness campaign assessments should be fetched.",
+          "description": "Specify the date until which phishing awareness campaign assessments should be fetched."
+        },
+        {
+          "name": "env",
+          "title": "Environment ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify the ID of the environment.",
+          "description": "Specify the ID of the environment. If an ID is not provided, the request will return latest report results for the Default environment."
+        }
+      ],
+      "output_schema": {
+        "success": "",
+        "cymulate": "",
+        "data": {
+          "attack": [],
+          "total": ""
+        }
+      }
+    },
+    {
+      "operation": "get_phishing_awareness_campaign_report_for_specific_assessment",
+      "title": "Get Phishing Awareness Campaign Report for Specific Assessment",
+      "annotation": "get_phishing_awareness_campaign_report_for_specific_assessment",
+      "description": "Retrieve Phishing Awareness Campaign report results for a specific assessment from Cymulate.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "id",
+          "title": "Assessment ID",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "Specify the assessment ID to fetch its phishing awareness campaign report.",
+          "description": "Specify the assessment ID to fetch its phishing awareness campaign report."
+        },
+        {
+          "name": "skip",
+          "title": "Offset",
+          "type": "integer",
+          "editable": true,
+          "visible": true,
+          "required": false,
+          "tooltip": "Specify the number of records to skip.",
+          "description": "Specify the number of records to skip."
```

Comment:
this description can be elaborated, you can refer this connector: cymulate-web-application-firewall